foreman/client.py

```diff
@@ -116,7 +116,8 @@ class ForemanVersionException(Exception):
 
 class MethodAPIDescription(object):
     exclude_html_reg = re.compile('</?[^>/]+/?>')
-    resource_pattern = re.compile(r'^/api(/v[12])?/(?P<resource>\w+).*')
+    resource_pattern = re.compile(
+        r'^/(\bapi\b|\bkatello/api\b)(/v[12])?/(?P<resource>\w+).*')
```

Comment:
Indeed, it is not needed. I forgot them there as I was doing a lot of tests to also include foreman-tasks/docker, but I dropped the support as there were not useful method for them.

It can be rewritten as:  r'^/(api|katello/api)(/v[12])?/(?P<resource>\w+).*')

Regards